main.py

```diff
@@ -1,18 +1,38 @@
 import os
-import lib.vtools_fetcher as vtf
+from lib.vtools_fetcher import EventFetcher
 from lib.calendar_api import CalendarAPI
 from dotenv import load_dotenv
 
 load_dotenv()
 
-country_id = os.getenv('COUNTRY_ID')
+vtf = EventFetcher(base_url=os.getenv('VTOOLS_API'))
+
+country_code = os.getenv('COUNTRY_CODE')
+
+# TODO: Create exception class for this error
+if country_code is None:
+    raise Exception()
+
+country_code = country_code.upper()
+country_id = None
+countries = vtf.fetch_countries()
+for country in countries:
+    id = country['id']
+    abbreviation = country['attributes']['abbreviation']
+    if abbreviation == country_code:
+        country_id = id
+        break
+
+# TODO: Create exception class for this error
+if country_id is None:
+    raise Exception()
```

Comment:
What I don't like about this is the inability to use the country_id instead. I would prefer to have `COUNTRY_CODE` as an override argument for it.